src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -293,6 +293,21 @@ export default {
     /** The Incorporation Application's Temporary Registration Number string. */
     tempRegNumber (): string {
       return sessionStorage.getItem('TEMP_REG_NUMBER')
+    },
+
+    displayName (): string {
```

Comment:
As this function does the same thing as EntityInfo.vue:nrTitle(), could you refactor these into a single function in, say, enum-mixin.ts ?

Reply:
Resolved!